annotations/views/relationtemplate_views.py

```diff
@@ -208,5 +206,12 @@ def create_from_relationtemplate(request, template_id):
     else:   # Not sure if we want to do anything for GET requests at this point.
         response_data = {}
 
-
     return JsonResponse(response_data)
+
+
+def delete_relationtemplate(request, template_id):
```

Comment:
this needs to be access restricted.

Reply:
Added "staff_member_required" before <b>delete_relationtemplate</b> method